.github/scripts/docker_compose.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+if [ -z "${ACS_VERSION}" ]; then
+    echo "ACS_VERSION variable is not setup "
+    exit 2;
+fi
+if [ -z "${COMMIT_MESSAGE}" ]; then
+    echo "COMMIT_MESSAGE variable is not setup "
+    exit 2;
+fi
+if [ -z "${GITHUB_REF}" ]; then
+    echo "GITHUB_REF variable is not setup "
+    exit 2;
+fi
+
+
+BRANCH_NAME=$(echo "${GITHUB_REF##*/}")
+GIT_DIFF=$(git diff origin/master --name-only .)
+compose_file="docker-compose.yml"
+alf_port=8080
+deploy=false
+
+if [[ ${ACS_VERSION} != "latest" ]]; then
+    export compose_file="${ACS_VERSION}-docker-compose.yml"
+fi
+if [[ "${BRANCH_NAME}" == "master" ]] || \
+   [[ "${COMMIT_MESSAGE}" == *"[run all tests]"* ]] || \
+   [[ "${COMMIT_MESSAGE}" == *"[release]"* ]] || \
+   [[ "${GIT_DIFF}" == *$compose_file* ]] || \
+   [[ "${GIT_DIFF}" == *test/postman/docker-compose* ]]
+then
+    deploy=true
+fi
+
+echo "Deploy:" "${deploy}"
+
+if ${deploy}; then
```

Comment:
indent the body otherwise is hard to tell which is the body of this if

not sure if this condition behave as intended, see https://stackoverflow.com/a/2953673/547195

if there is no `else` branch, just invert the condition and do an early exit